### PR TITLE
fence_cdu: add 8i support

### PR DESCRIPTION
**Pull request by watologo1**

### CODE REVIEW

File: agents/cdu/fence_cdu.py [hunk 1]
```diff
@@ -105,20 +142,17 @@ def main():
     docs["vendorurl"] = "http://www.servertech.com"
     show_docs(options, docs)
 
-    ## Support for --plug [switch]:[plug] notation that was used before
```
Comment (oalbrigt) [inline agents/cdu/fence_cdu.py:108]: We shouldn't remove parameters, as that might break setups on upgrade, or people following instructions where it's used.

**Comment by watologo1** [comment]
@oalbrigt Thanks for the quick review! I removed the separated patch which did remove the --switch option and did a:
git push ... --force
Should be better now?

**Review by oalbrigt — APPROVED** [review]

**Comment by oalbrigt** [comment]
ok to test

**Comment by oalbrigt** [comment]
You have to remove -- in `--api-version`, as that's only for CLI parameters, and will fail if it's not defined as an option that can be set with --api-parameter.

**Comment by oalbrigt** [comment]
retest this please

**Comment by oalbrigt** [comment]
Thanks.

**Pull request merged**